universal-application-tool-0.0.1/conf/evolutions/default/27.sql

```diff
@@ -0,0 +1,5 @@
+# --- !Ups
+alter table questions drop column path;
```

Comment:
Suggestion (non-blocking): please add `if exists` after `drop column` so it is less prone to error in unexpected situation.